libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -1022,18 +1024,16 @@ impl From<&Option<jsii::Docs>> for Docs {
 			return Docs::default()
 		};
 
-		let docs = docs.clone();
```

Comment:
Why remove the clone? I saw below that you removed the Clone derive on Docs, but why not keep it instead?

Reply:
The performance optimization was removing an unneeded clone of the entire assembly in `add_assembly`. To ensure we don't do any unneeded memory copies I figured just making sure the JSII structs aren't cloneable to begin with. This way we need to be explicit about what we copy out of these structs. Since we copy lots of data out of `Doc` then this might seem a bit tedious but still seemed worth it.